app/templates/breadcrumb.html

```diff
@@ -1,5 +1,9 @@
 <ol>
-  <% _.each(parents, function iterator(parent, key) { %><li>
-      <a href="#<%- parent.url %>"><%- parent.title %></a></li>
```

Comment:
If we use in templates <%- in template we don't need escape strings before that, so you don't need do this in view.